deny.toml

```diff
@@ -173,6 +173,7 @@ allow-registry = ["https://github.com/rust-lang/crates.io-index"]
 allow-git = [
     "https://github.com/apache/arrow-rs.git",
     "https://github.com/tikv/raft-rs.git",
+    "https://github.com/pcholakov/rust-rocksdb",
```

Comment:
Temporary, just for the PR checks to build - will remove after last PR is merged.